File: iOS/Objects/src/B4iProject/LS_frmteplate_slideshow.m

#import "iCore.h"
@interface LS_frmteplate_slideshow:NSObject
@end

@implementation LS_frmteplate_slideshow

- (void)LS_general:(B4ILayoutData*)views :(int)width :(int)height{
[B4ILayoutBuilder setScaleRate:0.3];
[B4ILayoutBuilder scaleAll:views :width :height];
[views get:@"imgads"].Height = (int)(([views get:@"pnlads"].Height));
[views get:@"imgads"].Width = (int)(([views get:@"pnlads"].Width));
[views get:@"panel1"].Left = (int)(([views get:@"pnlads"].Width)/2.0 - ([views get:@"panel1"].Width / 2));
[views get:@"panel1"].Top = (int)(([views get:@"pnlads"].Height)-([views get:@"panel1"].Height));
[views get:@"pbslideshow"].Left = (int)(([views get:@"pnlads"].Width)/2.0 - ([views get:@"pbslideshow"].Width / 2));
[views get:@"pbslideshow"].Top = (int)(([views get:@"pnlads"].Height)/2.0 - ([views get:@"pbslideshow"].Height / 2));

}
@end

File: Android/Objects/src/com/apps/demo/designerscripts/LS_frmteplate_slideshow.java
package com.apps.demo.designerscripts;
import anywheresoftware.b4a.objects.TextViewWrapper;
import anywheresoftware.b4a.objects.ImageViewWrapper;
import anywheresoftware.b4a.BA;


public class LS_frmteplate_slideshow{

public static void LS_general(java.util.LinkedHashMap<String, anywheresoftware.b4a.keywords.LayoutBuilder.ViewWrapperAndAnchor> views, int width, int height, float scale) {
anywheresoftware.b4a.keywords.LayoutBuilder.setScaleRate(0.3);
anywheresoftware.b4a.keywords.LayoutBuilder.scaleAll(views);
views.get("imgads").vw.setHeight((int)((views.get("pnlads").vw.getHeight())));
views.get("imgads").vw.setWidth((int)((views.get("pnlads").vw.getWidth())));
views.get("panel1").vw.setLeft((int)((views.get("pnlads").vw.getWidth())/2d - (views.get("panel1").vw.getWidth() / 2)));
views.get("panel1").vw.setTop((int)((views.get("pnlads").vw.getHeight())-(views.get("panel1").vw.getHeight())));
views.get("pbslideshow").vw.setLeft((int)((views.get("pnlads").vw.getWidth())/2d - (views.get("pbslideshow").vw.getWidth() / 2)));
views.get("pbslideshow").vw.setTop((int)((views.get("pnlads").vw.getHeight())/2d - (views.get("pbslideshow").vw.getHeight() / 2)));

}
}